src/main/kotlin/no/nav/familie/ef/sak/validering/BehandlingConstraint.kt

```diff
@@ -0,0 +1,14 @@
+package no.nav.familie.ef.sak.validering
+
+import javax.validation.Constraint
+import javax.validation.Payload
+import kotlin.reflect.KClass
+
+@Suppress("unused")
+@MustBeDocumented
+@Constraint(validatedBy = [Behandlingstilgang::class])
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.VALUE_PARAMETER)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class BehandlingConstraint(val message: String = "Ikke tilgang til behandling",
```

Comment:
Hvordan funker dette? :D

Reply:
Denne var en av de filene som egentligen hade en rename, men så endret vi samtidigt i fila så den fikk vel ikke meg seg att det var en rename. Tidligere SakConstraint.
I controllert kan man bruke @BehandlingConstraint for eks en parameter i metoden

Så må man ha en ConstraintValidator for hver type som skal valideres, eks UUID til behandlingId, så brukes Behandlingstilgang for å sjekke att man har tilgang. Hvis man ikke har tilgang blir det kastet ett styggt feil 😢

Reply:
aight - kanskje en feil som bør plukkes opp og retunere Ressurs.IKKE_TILGANG?

Reply:
Yep, tror dette er noe vi får se videre på senere :)